File: src/main/java/controller/user/femployeremployeereport.java
package controller.user;

import java.io.IOException;
import java.util.ArrayList;
import java.util.HashMap;

import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

import bean.FEmployerEmployeeBean;
import bean.FeedbackBean;
import bean.FeedbackReportBean;
import model.FeedbackDao2;

/**
 * Servlet implementation class femployeremployeereport
 */
@WebServlet("/dir1/femployeremployeereport")
public class femployeremployeereport extends HttpServlet {
	private static final long serialVersionUID = 1L;
	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {

		try
		{
			int fid=Integer.parseInt(request.getParameter("fid"));
			FeedbackDao2 dao=new FeedbackDao2();
			FeedbackBean feedback=dao.getFeedback(fid);
			request.setAttribute("feedback", feedback);
			HashMap<Integer, FeedbackReportBean> m=dao.getEmployerEmployeeReport(fid);

			if(m.get(1).getTotal()==0){
				request.getRequestDispatcher("femptyreport.jsp").forward(request, response);
			}else{

				ArrayList<FEmployerEmployeeBean> list=dao.getEmployerEmployeeDetailReport(fid);
				request.setAttribute("list", list);
                request.setAttribute("edtls", dao.getEmployerEmployeeDetails(fid));
				request.setAttribute("total", m.get(1).getTotal());

				double total=(double)m.get(1).getTotal();
				//Q1
				double q1a1=(double)m.get(1).getA1()/total;
				double q1a2=(double)m.get(1).getA2()/total;
				double q1a3=(double)m.get(1).getA3()/total;
				double q1a4=(double)m.get(1).getA4()/total;
				double q1a5=(double)m.get(1).getA5()/total;

				request.setAttribute("q1a1", q1a1);
				request.setAttribute("q1a2", q1a2);
				request.setAttribute("q1a3", q1a3);
				request.setAttribute("q1a4", q1a4);
				request.setAttribute("q1a5", q1a5);

				//Q2
				double q2a1=(double)m.get(2).getA1()/total;
				double q2a2=(double)m.get(2).getA2()/total;
				double q2a3=(double)m.get(2).getA3()/total;
				double q2a4=(double)m.get(2).getA4()/total;
				double q2a5=(double)m.get(2).getA5()/total;

				request.setAttribute("q2a1", q2a1);
				request.setAttribute("q2a2", q2a2);
				request.setAttribute("q2a3", q2a3);
				request.setAttribute("q2a4", q2a4);
				request.setAttribute("q2a5", q2a5);

				//Q3
				double q3a1=(double)m.get(3).getA1()/total;
				double q3a2=(double)m.get(3).getA2()/total;
				double q3a3=(double)m.get(3).getA3()/total;
				double q3a4=(double)m.get(3).getA4()/total;
				double q3a5=(double)m.get(3).getA5()/total;

				request.setAttribute("q3a1", q3a1);
				request.setAttribute("q3a2", q3a2);
				request.setAttribute("q3a3", q3a3);
				request.setAttribute("q3a4", q3a4);
				request.setAttribute("q3a5", q3a5);

				//Q4
				double q4a1=(double)m.get(4).getA1()/total;
				double q4a2=(double)m.get(4).getA2()/total;
				double q4a3=(double)m.get(4).getA3()/total;
				double q4a4=(double)m.get(4).getA4()/total;
				double q4a5=(double)m.get(4).getA5()/total;

				request.setAttribute("q4a1", q4a1);
				request.setAttribute("q4a2", q4a2);
				request.setAttribute("q4a3", q4a3);
				request.setAttribute("q4a4", q4a4);
				request.setAttribute("q4a5", q4a5);

				//Q5
				double q5a1=(double)m.get(5).getA1()/total;
				double q5a2=(double)m.get(5).getA2()/total;
				double q5a3=(double)m.get(5).getA3()/total;
				double q5a4=(double)m.get(5).getA4()/total;
				double q5a5=(double)m.get(5).getA5()/total;

				request.setAttribute("q5a1", q5a1);
				request.setAttribute("q5a2", q5a2);
				request.setAttribute("q5a3", q5a3);
				request.setAttribute("q5a4", q5a4);
				request.setAttribute("q5a5", q5a5);

				//Q6
				double q6a1=(double)m.get(6).getA1()/total;
				double q6a2=(double)m.get(6).getA2()/total;
				double q6a3=(double)m.get(6).getA3()/total;
				double q6a4=(double)m.get(6).getA4()/total;
				double q6a5=(double)m.get(6).getA5()/total;

				request.setAttribute("q6a1", q6a1);
				request.setAttribute("q6a2", q6a2);
				request.setAttribute("q6a3", q6a3);
				request.setAttribute("q6a4", q6a4);
				request.setAttribute("q6a5", q6a5);

				//Q7
				double q7a1=(double)m.get(7).getA1()/total;
				double q7a2=(double)m.get(7).getA2()/total;
				double q7a3=(double)m.get(7).getA3()/total;
				double q7a4=(double)m.get(7).getA4()/total;
				double q7a5=(double)m.get(7).getA5()/total;

				request.setAttribute("q7a1", q7a1);
				request.setAttribute("q7a2", q7a2);
				request.setAttribute("q7a3", q7a3);
				request.setAttribute("q7a4", q7a4);
				request.setAttribute("q7a5", q7a5);

				//Q8
				double q8a1=(double)m.get(8).getA1()/total;
				double q8a2=(double)m.get(8).getA2()/total;
				double q8a3=(double)m.get(8).getA3()/total;
				double q8a4=(double)m.get(8).getA4()/total;
				double q8a5=(double)m.get(8).getA5()/total;

				request.setAttribute("q8a1", q8a1);
				request.setAttribute("q8a2", q8a2);
				request.setAttribute("q8a3", q8a3);
				request.setAttribute("q8a4", q8a4);
				request.setAttribute("q8a5", q8a5);

				//Q9
				double q9a1=(double)m.get(9).getA1()/total;
				double q9a2=(double)m.get(9).getA2()/total;
				double q9a3=(double)m.get(9).getA3()/total;
				double q9a4=(double)m.get(9).getA4()/total;
				double q9a5=(double)m.get(9).getA5()/total;

				request.setAttribute("q9a1", q9a1);
				request.setAttribute("q9a2", q9a2);
				request.setAttribute("q9a3", q9a3);
				request.setAttribute("q9a4", q9a4);
				request.setAttribute("q9a5", q9a5);

				//Q10
				double q10a1=(double)m.get(10).getA1()/total;
				double q10a2=(double)m.get(10).getA2()/total;
				double q10a3=(double)m.get(10).getA3()/total;
				double q10a4=(double)m.get(10).getA4()/total;
				double q10a5=(double)m.get(10).getA5()/total;

				request.setAttribute("q10a1", q10a1);
				request.setAttribute("q10a2", q10a2);
				request.setAttribute("q10a3", q10a3);
				request.setAttribute("q10a4", q10a4);
				request.setAttribute("q10a5", q10a5);
				
				//Q11
				double q11a1=(double)m.get(11).getA1()/total;
				double q11a2=(double)m.get(11).getA2()/total;
				double q11a3=(double)m.get(11).getA3()/total;
				double q11a4=(double)m.get(11).getA4()/total;
				double q11a5=(double)m.get(11).getA5()/total;

				request.setAttribute("q11a1", q11a1);
				request.setAttribute("q11a2", q11a2);
				request.setAttribute("q11a3", q11a3);
				request.setAttribute("q11a4", q11a4);
				request.setAttribute("q11a5", q11a5);
				
				//Q12
				double q12a1=(double)m.get(12).getA1()/total;
				double q12a2=(double)m.get(12).getA2()/total;
				double q12a3=(double)m.get(12).getA3()/total;
				double q12a4=(double)m.get(12).getA4()/total;
				double q12a5=(double)m.get(12).getA5()/total;

				request.setAttribute("q12a1", q12a1);
				request.setAttribute("q12a2", q12a2);
				request.setAttribute("q12a3", q12a3);
				request.setAttribute("q12a4", q12a4);
				request.setAttribute("q12a5", q12a5);
				
				//Q13
				double q13a1=(double)m.get(13).getA1()/total;
				double q13a2=(double)m.get(13).getA2()/total;
				double q13a3=(double)m.get(13).getA3()/total;
				double q13a4=(double)m.get(13).getA4()/total;
				double q13a5=(double)m.get(13).getA5()/total;

				request.setAttribute("q13a1", q13a1);
				request.setAttribute("q13a2", q13a2);
				request.setAttribute("q13a3", q13a3);
				request.setAttribute("q13a4", q13a4);
				request.setAttribute("q13a5", q13a5);

				//Q11
//				DOUBLE Q11A1=(DOUBLE)M.GET(11).GETA1()/TOTAL;
//				DOUBLE Q11A2=(DOUBLE)M.GET(11).GETA2()/TOTAL;
//				DOUBLE Q11A3=(DOUBLE)M.GET(11).GETA3()/TOTAL;
//				DOUBLE Q11A4=(DOUBLE)M.GET(11).GETA4()/TOTAL;
//				DOUBLE Q11A5=(DOUBLE)M.GET(11).GETA5()/TOTAL;
//
//				REQUEST.SETATTRIBUTE("Q11A1", Q11A1);
//				REQUEST.SETATTRIBUTE("Q11A2", Q11A2);
//				REQUEST.SETATTRIBUTE("Q11A3", Q11A3);
//				REQUEST.SETATTRIBUTE("Q11A4", Q11A4);
//				REQUEST.SETATTRIBUTE("Q11A5", Q11A5);

				//Q12
//				double q12a1=(double)m.get(12).getA1()/total;
//				double q12a2=(double)m.get(12).getA2()/total;
//				double q12a3=(double)m.get(12).getA3()/total;
//				double q12a4=(double)m.get(12).getA4()/total;
//				double q12a5=(double)m.get(12).getA5()/total;
//
//				request.setAttribute("q12a1", q12a1);
//				request.setAttribute("q12a2", q12a2);
//				request.setAttribute("q12a3", q12a3);
//				request.setAttribute("q12a4", q12a4);
//				request.setAttribute("q12a5", q12a5);

				//Q13
//				double q13a1=(double)m.get(13).getA1()/total;
//				double q13a2=(double)m.get(13).getA2()/total;
//				double q13a3=(double)m.get(13).getA3()/total;
//				double q13a4=(double)m.get(13).getA4()/total;
//				double q13a5=(double)m.get(13).getA5()/total;
//
//				request.setAttribute("q13a1", q13a1);
//				request.setAttribute("q13a2", q13a2);
//				request.setAttribute("q13a3", q13a3);
//				request.setAttribute("q13a4", q13a4);
//				request.setAttribute("q13a5", q13a5);

		    	request.getRequestDispatcher("femployeremployeereport.jsp").forward(request, response);
			}


		}
		catch(Exception e)
		{
			e.printStackTrace();
			response.sendRedirect("femployeremployeereport.jsp?success=3");
		}

	}

}

File: src/main/webapp/js/piechart/femployeremployeereport.js
$(document).ready(function(){
			
				google.charts.load('current', {'packages':['corechart']});
				google.charts.setOnLoadCallback(drawChart);

				function drawChart() {

				    // q1
				    var q1 = new google.visualization.DataTable();
				    q1.addColumn('string', 'Topping');
				    q1.addColumn('number', 'Slices');
				    q1.addRows([
				      ['Far from Satisfied', parseFloat($("#q1a1").val())],
				      ['Poor', parseFloat($("#q1a2").val())],
				      ['Satisfactory', parseFloat($("#q1a3").val())],
				      ['Good', parseFloat($("#q1a4").val())],
				      ['Excellent', parseFloat($("#q1a5").val())]
				    ]);

				    var q1o = {'title':'1. Ability to contribute to the goal of the organization',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    var q1chart = new google.visualization.PieChart(document.getElementById('q1'));
				    q1chart.draw(q1, q1o);
				    
				    //q2
				    var q2 = new google.visualization.DataTable();
				    q2.addColumn('string', 'Topping');
				    q2.addColumn('number', 'Slices');
				    q2.addRows([
				      ['Far from Satisfied', parseFloat($("#q2a1").val())],
				      ['Poor', parseFloat($("#q2a2").val())],
				      ['Satisfactory', parseFloat($("#q2a3").val())],
				      ['Good', parseFloat($("#q2a4").val())],
				      ['Excellent', parseFloat($("#q2a5").val())]
				    ]);

				    // Set chart options
				    var q2o = {'title':'2. Planning and organization skill',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q2chart = new google.visualization.PieChart(document.getElementById('q2'));
				    q2chart.draw(q2, q2o);
				    
				    //q3
				    var q3 = new google.visualization.DataTable();
				    q3.addColumn('string', 'Topping');
				    q3.addColumn('number', 'Slices');
				    q3.addRows([
				      ['Far from Satisfied', parseFloat($("#q3a1").val())],
				      ['Poor', parseFloat($("#q3a2").val())],
				      ['Satisfactory', parseFloat($("#q3a3").val())],
				      ['Good', parseFloat($("#q3a4").val())],
				      ['Excellent', parseFloat($("#q3a5").val())]
				    ]);

				    // Set chart options
				    var q3o = {'title':'3. Communication skill and Soft skill',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q3chart = new google.visualization.PieChart(document.getElementById('q3'));
				    q3chart.draw(q3, q3o);
				    
				  //q4
				    var q4 = new google.visualization.DataTable();
				    q4.addColumn('string', 'Topping');
				    q4.addColumn('number', 'Slices');
				    q4.addRows([
				      ['Far from Satisfied', parseFloat($("#q4a1").val())],
				      ['Poor', parseFloat($("#q4a2").val())],
				      ['Satisfactory', parseFloat($("#q4a3").val())],
				      ['Good', parseFloat($("#q4a4").val())],
				      ['Excellent', parseFloat($("#q4a5").val())]
				    ]);

				    // Set chart options
				    var q4o = {'title':'4. Relationship with Seniors and Subordinates',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q4chart = new google.visualization.PieChart(document.getElementById('q4'));
				    q4chart.draw(q4, q4o);
				    
				  //q5
				    var q5 = new google.visualization.DataTable();
				    q5.addColumn('string', 'Topping');
				    q5.addColumn('number', 'Slices');
				    q5.addRows([
				      ['Far from Satisfied', parseFloat($("#q5a1").val())],
				      ['Poor', parseFloat($("#q5a2").val())],
				      ['Satisfactory', parseFloat($("#q5a3").val())],
				      ['Good', parseFloat($("#q5a4").val())],
				      ['Excellent', parseFloat($("#q5a5").val())]
				    ]);

				    // Set chart options
				    var q5o = {'title':'5. Leadership, Team spirit and initiative',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q5chart = new google.visualization.PieChart(document.getElementById('q5'));
				    q5chart.draw(q5, q5o);
				    
				  //q6
				    var q6 = new google.visualization.DataTable();
				    q6.addColumn('string', 'Topping');
				    q6.addColumn('number', 'Slices');
				    q6.addRows([
				      ['Far from Satisfied', parseFloat($("#q6a1").val())],
				      ['Poor', parseFloat($("#q6a2").val())],
				      ['Satisfactory', parseFloat($("#q6a3").val())],
				      ['Good', parseFloat($("#q6a4").val())],
				      ['Excellent', parseFloat($("#q6a5").val())]
				    ]);

				    // Set chart options
				    var q6o = {'title':'6. Willingness to learn new techniques, adopt new ideas etc',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q6chart = new google.visualization.PieChart(document.getElementById('q6'));
				    q6chart.draw(q6, q6o);
				    
				  //q7
				    var q7 = new google.visualization.DataTable();
				    q7.addColumn('string', 'Topping');
				    q7.addColumn('number', 'Slices');
				    q7.addRows([
				      ['Far from Satisfied', parseFloat($("#q7a1").val())],
				      ['Poor', parseFloat($("#q7a2").val())],
				      ['Satisfactory', parseFloat($("#q7a3").val())],
				      ['Good', parseFloat($("#q7a4").val())],
				      ['Excellent', parseFloat($("#q7a5").val())]
				    ]);

				    // Set chart options
				    var q7o = {'title':'7. Ability to use work place equipment',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q7chart = new google.visualization.PieChart(document.getElementById('q7'));
				    q7chart.draw(q7, q7o);
				    
				    
				  //q8
				    var q8 = new google.visualization.DataTable();
				    q8.addColumn('string', 'Topping');
				    q8.addColumn('number', 'Slices');
				    q8.addRows([
				      ['Far from Satisfied', parseFloat($("#q8a1").val())],
				      ['Poor', parseFloat($("#q8a2").val())],
				      ['Satisfactory', parseFloat($("#q8a3").val())],
				      ['Good', parseFloat($("#q8a4").val())],
				      ['Excellent', parseFloat($("#q8a5").val())]
				    ]);

				    // Set chart options
				    var q8o = {'title':'8. Ability to solve work place problems',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q8chart = new google.visualization.PieChart(document.getElementById('q8'));
				    q8chart.draw(q8, q8o);
				    
				  //q9
				    var q9 = new google.visualization.DataTable();
				    q9.addColumn('string', 'Topping');
				    q9.addColumn('number', 'Slices');
				    q9.addRows([
				      ['Far from Satisfied', parseFloat($("#q9a1").val())],
				      ['Poor', parseFloat($("#q9a2").val())],
				      ['Satisfactory', parseFloat($("#q9a3").val())],
				      ['Good', parseFloat($("#q9a4").val())],
				      ['Excellent', parseFloat($("#q9a5").val())]
				    ]);

				    // Set chart options
				    var q9o = {'title':'9. Innovativeness , creativity',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q9chart = new google.visualization.PieChart(document.getElementById('q9'));
				    q9chart.draw(q9, q9o);
				    
				  //q10
				    var q10 = new google.visualization.DataTable();
				    q10.addColumn('string', 'Topping');
				    q10.addColumn('number', 'Slices');
				    q10.addRows([
				      ['Far from Satisfied', parseFloat($("#q10a1").val())],
				      ['Poor', parseFloat($("#q10a2").val())],
				      ['Satisfactory', parseFloat($("#q10a3").val())],
				      ['Good', parseFloat($("#q10a4").val())],
				      ['Excellent', parseFloat($("#q10a5").val())]
				    ]);

				    // Set chart options
				    var q10o = {'title':'10. Involvement in social activities',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q10chart = new google.visualization.PieChart(document.getElementById('q10'));
				    q10chart.draw(q10, q10o);
				    
				    //q11
				    var q11 = new google.visualization.DataTable();
				    q11.addColumn('string', 'Topping');
				    q11.addColumn('number', 'Slices');
				    q11.addRows([
				      ['Far from Satisfied', parseFloat($("#q11a1").val())],
				      ['Poor', parseFloat($("#q11a2").val())],
				      ['Satisfactory', parseFloat($("#q11a3").val())],
				      ['Good', parseFloat($("#q11a4").val())],
				      ['Excellent', parseFloat($("#q11a5").val())]
				    ]);

				    // Set chart options
				    var  q11o = {'title':'11. Emphasis is on both fundamentals as well as latest developments',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var  q11chart = new google.visualization.PieChart(document.getElementById('q11'));
				    q11chart.draw(q11, q11o);
				    
				      //q12
				    var q12 = new google.visualization.DataTable();
				    q12.addColumn('string', 'Topping');
				    q12.addColumn('number', 'Slices');
				    q12.addRows([
				      ['Far from Satisfied', parseFloat($("#q12a1").val())],
				      ['Poor', parseFloat($("#q12a2").val())],
				      ['Satisfactory', parseFloat($("#q12a3").val())],
				      ['Good', parseFloat($("#q12a4").val())],
				      ['Excellent', parseFloat($("#q12a5").val())]
				    ]);

				    // Set chart options
				    var  q12o = {'title':'12. Imparts knowledge and skills required for the job market',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var  q12chart = new google.visualization.PieChart(document.getElementById('q12'));
				    q12chart.draw(q12, q12o);
				    
				    //q13
				    var q13 = new google.visualization.DataTable();
				    q13.addColumn('string', 'Topping');
				    q13.addColumn('number', 'Slices');
				    q13.addRows([
				      ['Far from Satisfied', parseFloat($("#q13a1").val())],
				      ['Poor', parseFloat($("#q13a2").val())],
				      ['Satisfactory', parseFloat($("#q13a3").val())],
				      ['Good', parseFloat($("#q13a4").val())],
				      ['Excellent', parseFloat($("#q13a5").val())]
				    ]);

				    // Set chart options
				    var  q13o = {'title':'13. Incorporates computer skills and other soft skills needed for employment.',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var  q13chart = new google.visualization.PieChart(document.getElementById('q13'));
				    q13chart.draw(q13, q13o);
				    
				  /*//q11
				    var q11 = new google.visualization.DataTable();
				    q11.addColumn('string', 'Topping');
				    q11.addColumn('number', 'Slices');
				    q11.addRows([
				      ['Far from Satisfied', parseFloat($("#q11a1").val())],
				      ['Not Satisfied', parseFloat($("#q11a2").val())],
				      ['Satisfied', parseFloat($("#q11a3").val())],
				      ['Happy', parseFloat($("#q11a4").val())],
				      ['Very Happy', parseFloat($("#q11a5").val())]
				    ]);

				    // Set chart options
				    var q11o = {'title':'11. Involvement in social activities',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q11chart = new google.visualization.PieChart(document.getElementById('q11'));
				    q11chart.draw(q11, q11o);
				    
				  //q12
				    var q12 = new google.visualization.DataTable();
				    q12.addColumn('string', 'Topping');
				    q12.addColumn('number', 'Slices');
				    q12.addRows([
				      ['Far from Satisfied', parseFloat($("#q12a1").val())],
				      ['Not Satisfied', parseFloat($("#q12a2").val())],
				      ['Satisfied', parseFloat($("#q12a3").val())],
				      ['Happy', parseFloat($("#q12a4").val())],
				      ['Very Happy', parseFloat($("#q12a5").val())]
				    ]);

				    // Set chart options
				    var q12o = {'title':'12. Simplicity and sense of belonging',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q12chart = new google.visualization.PieChart(document.getElementById('q12'));
				    q12chart.draw(q12, q12o);
				    
				    
				  //q13
				    var q13 = new google.visualization.DataTable();
				    q13.addColumn('string', 'Topping');
				    q13.addColumn('number', 'Slices');
				    q13.addRows([
				      ['Far from Satisfied', parseFloat($("#q13a1").val())],
				      ['Not Satisfied', parseFloat($("#q13a2").val())],
				      ['Satisfied', parseFloat($("#q13a3").val())],
				      ['Happy', parseFloat($("#q13a4").val())],
				      ['Very Happy', parseFloat($("#q13a5").val())]
				    ]);

				    // Set chart options
				    var q13o = {'title':'13. Respect for values in life',
				    		'is3D': true, 
				    		'tooltip': {'text': 'percentage'},
				    		'legend': { 'position': 'labeled' },
				    		'height':300};

				    // Instantiate and draw our chart, passing in some options.
				    var q13chart = new google.visualization.PieChart(document.getElementById('q13'));
				    q13chart.draw(q13, q13o);*/
				  }

				//Print Button Onclick event to print Pie charts table 
        		$("#printBtn").click(function(event){
        			w=window.open('', '#printBlock', 'scrollbars=yes'); 
        			var myStyle = '<link rel="stylesheet" type="text/css" href="../../css/style.css" />'
        		        +'<link rel="stylesheet" type="text/css" href="../../css/form.css" />'
        		        +'<link rel="stylesheet" type="text/css" href="../../css/table.css"/>'
        		        +'<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">'
        		        +'<style type="text/css">'
        		        +'@media print {'
        		        +'	body {-webkit-print-color-adjust: exact;}'
        		        +'table { page-break-after:auto }'
        		        +'table tr { page-break-inside:avoid; page-break-after:auto }'
        		        +'table tr td { page-break-inside:avoid; page-break-after:auto }'
        		        +'table thead { display:table-header-group }'
        		        +'table tr.lstrow { page-break-inside:avoid; page-break-after:auto }'
        		        +'}'
        		        +'table#chartTbl tr td{'
        		        +'height: 60px;'
        		        +'}'
        		        +'</style>';
        			w.document.write(myStyle + jQuery('#printBlock').html());
        			w.document.write('<scr' + 'ipt type="text/javascript">' + 'window.onload = function() { window.print(); window.close(); };' + '</sc' + 'ript>');

        		    w.document.close(); // necessary for IE >= 10
        		    w.focus();
        		});		
});